docs/tasks/step-4.md

```diff
@@ -1,30 +1,129 @@
 
-When you check or uncheck a checkbox these selected items will change. You should use the `updateBook` action in the Pinia `bookStore` to update the books by setting and un-setting the favorites field when a book is checked and unchecked on the table.
+### Example using HTML tags
+Based on the example above, I've styled each type of element with some basic CSS. Any colors (e.g. `background-color`, `color`) can be styled in a few different ways. Don't forget everything is done in American English.
+1. Color name, e.g. `gray` or `greenyellow`.
+2. Hex color code, e.g. `#E2F9B8` or `#D16014`.
+3. RGB, e.g. `rgb(238, 222, 240)` or `rgb(70, 3, 28)`.
 
-## b) Unfavorite a book from the homepage
+A hex code is a six-digit combination of numbers and letters defined by its mix of red, green and blue (RGB) for a color.</br>
+RGB is a model where the red, green and blue primary colors are added together in various ways to reproduce a broad array of colors.
 
-For each book card component in the list of favorite books on the homepage, add a button so that you can unfavorite the book. Use the `updateBook` action again and you can use a vuetify button component if you wish (https://vuetifyjs.com/en/components/buttons/).
+``` css
+h1 {
+    background-color: gray;  /* uses color name */
+    color: greenyellow;
+    font-family: Arial, sans-serif; /* tells the browser which font to use */
+    padding: 25px; /* tells the browser how much padding to put around the element */
+}
 
-## c) Favorite a book from the homepage
+h2 {
+    background-color: #E2F9B8; /* uses hex code */
+    color: #D16014;
+    border-radius: 15px; /* tells the browser how rounded you want the element's box to be */
+    text-align: center; 
+    font-size: 20px; /* uses pixels for the text size */
+    max-width: 600px; /* tells the browser this element should only go up to 600px wide - it can go smaller if you decrease your screen size */
+    padding: 5px;
+}
 
-Add a button to the homepage that opens a drawer or modal that allows you to favorite a book.
+p {
+    background-color: rgb(238, 222, 240); /* uses RGB code */
+    color: rgb(70, 3, 28);
+    font-size: small; /* you can also use words for text size */
+    height: 200px; /* will always display at this height */
+}
+```
+This set of CSS rules will be applied to our headers and paragraphs as so:
 
-## d) Edit a books details
+![CSS using html tags](./css-tags.png)
 
-Add a way for a user to edit a books details (from the homepage and from the table). Use the `updateBook` action in the store. Include a way to remove a book from your library list using the `deleteBook` store action.
+Have a look through [the w3schools CSS docs](https://www.w3schools.com/css/css_intro.asp) for some more info and quick practice sessions.
+
+## b) Using classes and ids
+As well as using HTML tags to style specific types of element, we can give a name to any set of elements, called a class or an id.</br>
```

Comment:
Might need to separate this out as it looks like I'm saying the id can also be a set of elements...